optimum/intel/neural_compressor/trainer.py

```diff
@@ -630,3 +630,20 @@ def compute_distillation_loss(self, student_outputs, teacher_outputs):
             distillation_loss = loss if distillation_loss is None else distillation_loss + loss
         distillation_loss *= self.agent.criterion.temperature**2
         return distillation_loss
+
+    def evaluate(
+        self,
+        eval_dataset: Optional[Dataset] = None,
+        ignore_keys: Optional[List[str]] = None,
+        metric_key_prefix: str = "eval",
+    ) -> Dict[str, float]:
+        if self.model is None:
+            return None
+        if self.model.config and self.model.config.torch_dtype == "int8":

```

Comment:
I would replace it with :
```suggestion
        if getattr(self.model.config, "torch_dtype ", None) == "int8":
```